OpenDreamRuntime/Objects/DreamObject.cs

```diff
@@ -6,13 +6,15 @@
 namespace OpenDreamRuntime.Objects {
     [Virtual]
     public class DreamObject {
+        [Dependency] private readonly IDreamManager _dreamMan = default!;
         public DreamObjectDefinition? ObjectDefinition { get; protected set; }
         public bool Deleted = false;
 
         private Dictionary<string, DreamValue> _variables = new();
 
         public DreamObject(DreamObjectDefinition? objectDefinition) {
             ObjectDefinition = objectDefinition;
+            IoCManager.InjectDependencies(this);
```

Comment:
How performant is this?

Reply:
DreamObject's constructor isn't even showing up on the profiler. Plus it should cache the injector after the first time.